services/jwt.js

```diff
@@ -120,6 +120,11 @@ function SharedSecret({ kid = undefined, secret = null }, algorithm) {
     throw new Error('Secret cannot have a zero length');
   }
 
+  // If the secret is base64 encoded, then decode it!
+  if (secret.startsWith('base64:')) {
+    secret = Buffer.from(secret.substring(6), 'base64').toString();
```

Comment:
![image](https://user-images.githubusercontent.com/14221600/47220483-30bafb80-d3b2-11e8-969a-a9c7feb067b7.png)

Shouldn't it be `secret.substring(7)`?